src/docs/composing_applications.md

```diff
@@ -104,11 +104,17 @@ An example `package.json` may look like the following:
     "vscode.git",
     "vscode.git-ui",
     "vscode.github",
-    "vscode.markdown-language-features"
+    "vscode.markdown-language-features",
+    "vscode.microsoft-authentication"
   ]
 }
 ```
 
+The following properties are used to consume plugins at builtins (bundled extensions):
```

Comment:
This sounds a bit strange: "[...] to consume plugins __at builtins__". I think "to consume plugins" might be enough?

Reply:
"to consume built-in plugins"?

Reply:
updated :+1: